tools/autograd/derivatives.yaml

```diff
@@ -443,6 +445,7 @@
 
 - name: deg2rad(Tensor self) -> Tensor
   self: deg2rad_backward(grad)
+  result: auto_element_wise
```

Comment:
It's probably too late for this, but in the future, if there are more auto_element_wise and auto_linear formulas, if you separate them out from the formulas that need more thinking about we can review and ship them faster

Reply:
Sure!